examples/RTR-exampleChromeDriver/tests/home.js

```diff
@@ -6,39 +6,36 @@ describe(suiteName, function() {
 
   this.retries(3);
 
-  before((browser, done) => {
+  before(() => {
     const item = {
       name: suiteName,
-      attributes: [{ key: 'suite', value: 'home' }],
-      description: 'Common suite description',
+      attributes: [{ key: 'suite', value: 'home.js' }],
```

Comment:
Could you please tell me why you added the file format? Does it carry some meaning?

Reply:
It's just an example. So it's not important, I think :)